src/calendarUtils.ts

```diff
@@ -272,6 +274,9 @@ export const getMonthView: Function = ({events = [], viewDate, weekStartsOn}:
   const days: MonthViewDay[] = [];
   for (let i: number = 0; i < differenceInDays(end, start) + 1; i++) {
     const date: Date = addDays(start, i);
+    if (excluded && excluded.some(e => date.getDay() === e)) {
+      continue;
```

Comment:
I'm not a huge fan of using `continue` in loops as it can sometimes make the logic hard to follow, can you inver the logic and add the contents below into the if statement?